app/models.py

```diff
@@ -550,3 +557,21 @@ def bulk_lookup():
                 db.session.commit()
             except IntegrityError:
                 db.session.rollback()
+
+class Leaders(db.Model):
```

Comment:
I'm puzzled why you add Leaders etc as tables here, then drop the table and create a 'leaders' view instead